tools/packaging/scripts/lib.sh

```diff
@@ -132,6 +132,11 @@ push_to_registry() {
 	fi
 }
 
+get_initramfs_image_name() {
```

Comment:
This change and all the other related to initrd belong to the initrd commit, no?

Reply:
This is required to build the SEV kernel.

Reply:
@arronwy could you help here? In case I misunderstood this completely.... :D

Reply:
@UnmeshDeodhar I just realized that @arronwy has the https://github.com/kata-containers/kata-containers/pull/6756 opened to port the measured boot image/rootfs feature from CCv0 to main. So please remove all this initramfs related code from this PR.

Reply:
Thanks @wainersm for letting me know. I have removed the initramfs related changes now.